iaso/api/mobile/groups.py

```diff
@@ -0,0 +1,66 @@
+from django.db.models.query import QuerySet
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import permissions, serializers
+from rest_framework.generics import get_object_or_404
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from iaso.api.common import ModelViewSet
+from iaso.api.query_params import APP_ID
+from iaso.api.serializers import AppIdSerializer
+from iaso.models import Group, Project
+
+
+class MobileGroupSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Group
+        fields = [
+            "id",
+            "name",
+        ]
+
+
+class MobileGroupsViewSet(ModelViewSet):
+    """Groups API for Mobile.
+
+    Allows to retrieve a list of groups from the API.
+
+    Returns a lighter payload adapted for the mobile application.
+
+    `GET /api/mobile/groups/?app_id=some.app.id`
+    """
+
+    http_method_names = ["get", "head", "options"]
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
```

Comment:
Isn't that unnecessary?

Reply:
Do we really want an open endpoint here? Or do we want the same perms [as the other endoint](https://github.com/BLSQ/iaso/blob/00f13e2ba3fbdb92db340c0a26d02d0657e38837/iaso/api/groups.py#L92-L96)?

Reply:
Since you already limited the methods to `get`, `head`, and `options`. 
There are no writing methods allowed. Therefore, no permission here is the same as `IsAuthenticatedOrReadOnly`, isn't it?